*** progmodlinux.tex	Sat May 18 13:15:25 2002
--- progmodlinux_fix.tex	Sat May 18 13:15:45 2002
***************
*** 46,52 ****
  
  % El estilo del documento
  %\documentstyle[times,indentfirst,epsfig,twoside,linuxdoc,lotex]{report}
! \documentclass[times,indentfirst,epsfig,a4paper,linuxdoc,twoside,lotex]{report}
  
  % Cabeceras en Español
  \usepackage[spanish]{babel}
--- 46,52 ----
  
  % El estilo del documento
  %\documentstyle[times,indentfirst,epsfig,twoside,linuxdoc,lotex]{report}
! \documentclass[times,indentfirst,epsfig,linuxdoc,twoside,lotex]{report}
  
  % Cabeceras en Español
  \usepackage[spanish]{babel}
***************
*** 67,74 ****
  %\setlength\paperheight {9in}
  
  % Tabla de contenidos
! \setcounter{secnumdepth}{5}
! \setcounter{tocdepth}{2}
  
  % Inicialmente, númeración romana sin números.
  \pagenumbering{roman}
--- 67,74 ----
  %\setlength\paperheight {9in}
  
  % Tabla de contenidos
! %\setcounter{secnumdepth}{5}
! %\setcounter{tocdepth}{2}
  
  % Inicialmente, númeración romana sin números.
  \pagenumbering{roman}
***************
*** 392,398 ****
  My gratitud a Linus Torvalds, Richard Stallman y toda la otra gente que
  me hizo posible ejecutar un sistema operativo de  calidad en mi
  computadora y obtener el código fuente sin decirlo (si, correcto ---
! ¿entonces por qué lo dije?).
  
  \subsection{Para la versión 1.0.1}
  
--- 392,398 ----
  My gratitud a Linus Torvalds, Richard Stallman y toda la otra gente que
  me hizo posible ejecutar un sistema operativo de  calidad en mi
  computadora y obtener el código fuente sin decirlo (si, correcto ---
! ?`entonces por qué lo dije?).
  
  \subsection{Para la versión 1.0.1}
  
***************
*** 579,585 ****
  \end{verbatim}
  
  
! Entonces, ahora la única cosa que queda es hacer {\tt su} a root (¿no compilaste
  como root, verdad?) Viviendo en el límite\footnote{El motivo por el que prefiero no
  compilarlo como root es que cuanto menor sea hecho como root más seguro estará el 
  equipo. Yo trabajo en seguridad informática, por lo tanto estoy paranoico}\ldots), y
--- 579,585 ----
  \end{verbatim}
  
  
! Entonces, ahora la única cosa que queda es hacer {\tt su} a root (?`no compilaste
  como root, verdad?) Viviendo en el límite\footnote{El motivo por el que prefiero no
  compilarlo como root es que cuanto menor sea hecho como root más seguro estará el 
  equipo. Yo trabajo en seguridad informática, por lo tanto estoy paranoico}\ldots), y
***************
*** 2748,2754 ****
  definidos del núcleo para registrar archivos {\tt proc} y manejadores de 
  dispositivos. Esto está bien si quieres hacer algo a través de lo que
  los programadores del núcleo quisieran, tal como escribir un 
! controlador de dispositivo. Pero, ¿y si quieres escribir algo inusual, para
  cambiar el comportamiento del sistema de alguna forma? Entonces, te
  encuentras sólo.
  
--- 2748,2754 ----
  definidos del núcleo para registrar archivos {\tt proc} y manejadores de 
  dispositivos. Esto está bien si quieres hacer algo a través de lo que
  los programadores del núcleo quisieran, tal como escribir un 
! controlador de dispositivo. Pero, ?`y si quieres escribir algo inusual, para
  cambiar el comportamiento del sistema de alguna forma? Entonces, te
  encuentras sólo.
  
***************
*** 3066,3072 ****
  \index{procesos bloqueantes}
  \index{bloqueantes\\procesos}
  
! ¿Qué puedes hacer cuando alquien te pregunta por algo que no puedes hacer en el acto?
  Si eres un humano y estás siendo molestado por un humano, la única cosa
  que puedes decir es: `Ahora no. Estoy ocupado. {\em !`Vete!}''. Pero si eres
  un módulo del núcleo y estás siendo molestado por un proceso, tienes otra
--- 3066,3072 ----
  \index{procesos bloqueantes}
  \index{bloqueantes\\procesos}
  
! ?`Qué puedes hacer cuando alquien te pregunta por algo que no puedes hacer en el acto?
  Si eres un humano y estás siendo molestado por un humano, la única cosa
  que puedes decir es: `Ahora no. Estoy ocupado. {\em !`Vete!}''. Pero si eres
  un módulo del núcleo y estás siendo molestado por un proceso, tienes otra
***************
*** 4397,4403 ****
  
  
  
! \chapter{¿Desde aquí hasta dónde?}\label{where-to}
  
  Yo, fácilmente podría haber introducido unos pocos capítulos más en este
  libro. Podría haber añadido un capítulo sobre cómo crear nuevos sistemas de
--- 4397,4403 ----
  
  
  
! \chapter{?`Desde aquí hasta dónde?}\label{where-to}
  
  Yo, fácilmente podría haber introducido unos pocos capítulos más en este
  libro. Podría haber añadido un capítulo sobre cómo crear nuevos sistemas de
